Remove const qualifier from struct file_operations prior to 2.6.17

This eliminates the compiler warnings:
uverbs_main.c: In function `ib_uverbs_alloc_event_file':
uverbs_main.c:566: warning: assignment discards qualifiers from pointer target type
uverbs_main.c: In function `ib_uverbs_add_one':
uverbs_main.c:802: warning: assignment discards qualifiers from pointer target type

Signed-off-by: John Gregor <john.gregor@qlogic.com>

---
 drivers/infiniband/core/uverbs_main.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: ofed_kernel/drivers/infiniband/core/uverbs_main.c
===================================================================
--- ofed_kernel.orig/drivers/infiniband/core/uverbs_main.c
+++ ofed_kernel/drivers/infiniband/core/uverbs_main.c
@@ -400,7 +400,7 @@ static int ib_uverbs_event_close(struct 
 	return 0;
 }
 
-static const struct file_operations uverbs_event_fops = {
+static struct file_operations uverbs_event_fops = {
 	.owner	 = THIS_MODULE,
 	.read 	 = ib_uverbs_event_read,
 	.poll    = ib_uverbs_event_poll,
@@ -721,14 +721,14 @@ static int ib_uverbs_close(struct inode 
 	return 0;
 }
 
-static const struct file_operations uverbs_fops = {
+static struct file_operations uverbs_fops = {
 	.owner 	 = THIS_MODULE,
 	.write 	 = ib_uverbs_write,
 	.open 	 = ib_uverbs_open,
 	.release = ib_uverbs_close
 };
 
-static const struct file_operations uverbs_mmap_fops = {
+static struct file_operations uverbs_mmap_fops = {
 	.owner 	 = THIS_MODULE,
 	.write 	 = ib_uverbs_write,
 	.mmap    = ib_uverbs_mmap,
